rm/rm-server/src/test/java/functionaltests/StandardTestSuite.java

```diff
@@ -0,0 +1,94 @@
+import functionaltests.authentication.ConnectionTest;
+import functionaltests.authentication.ConnectionTest2;
+import functionaltests.dataspace.DataSpaceNodeConfigurationAgentTest;
+import functionaltests.db.NodeHistoryTest;
+import functionaltests.db.NodeSourcesTest;
+import functionaltests.db.RMDBManagerBufferTest;
+import functionaltests.db.RMDBManagerTest;
+import functionaltests.execremote.TestExecRemote;
+import functionaltests.jmx.RMProxyUserInterfaceTest;
+import functionaltests.jmx.ResourceManagerJMXTest;
+import functionaltests.jmx.account.AddGetDownRemoveTest;
+import functionaltests.jmx.account.AddGetReleaseRemoveTest;
+import functionaltests.jmx.account.AddGetRemoveTest;
+import functionaltests.jmx.account.AddGetTest;
+import functionaltests.monitor.TestRMMonitoring;
+import functionaltests.monitor.TestRMNodeMonitoring;
+import functionaltests.nodesource.TestLocalInfrastructureRestartDownNodesPolicy;
+import functionaltests.nodesource.TestLocalInfrastructureStaticPolicy;
+import functionaltests.nodesource.TestLocalInfrastructureTimeSlotPolicy;
+import functionaltests.nodesource.TestNodeSourceAfterRestart;
+import functionaltests.nodesource.TestSSHInfrastructureV2;
+import functionaltests.nodesource.TestSSHInfrastructureV2RestartDownNodesPolicy;
+import functionaltests.nodestate.TestAddRemoveAll;
+import functionaltests.nodestate.TestAdminAddingNodes;
+import functionaltests.nodestate.TestConcurrentUsers;
+import functionaltests.nodestate.TestNodeEncoding;
+import functionaltests.nodestate.TestNodeSourcesActions;
+import functionaltests.nodestate.TestNodesStates;
+import functionaltests.permissions.TestNSAdminPermissions;
+import functionaltests.permissions.TestNSProviderPermissions;
+import functionaltests.selectionscript.DynamicSelectionScriptTest;
+import functionaltests.selectionscript.SelectionWithNodesExclusionTest;
+import functionaltests.selectionscript.SelectionWithSeveralScriptsTest;
+import functionaltests.selectionscript.SelectionWithSeveralScriptsTest2;
+import functionaltests.selectionscript.StaticSelectionScriptTest;
+import functionaltests.selectionscript.UnauthorizedSelectionScriptTest;
+import functionaltests.topology.LocalSelectionTest;
+import functionaltests.topology.SelectionTest;
+
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({ AuthenticationTest.class, ConnectionTest.class, ConnectionTest2.class,
+                      DataSpaceNodeConfigurationAgentTest.class, NodeHistoryTest.class, NodeSourcesTest.class,
+                      RMDBManagerBufferTest.class, RMDBManagerTest.class, TestExecRemote.class,
+                      AddGetDownRemoveTest.class, AddGetReleaseRemoveTest.class, AddGetRemoveTest.class,
+                      AddGetTest.class, ResourceManagerJMXTest.class, RMProxyUserInterfaceTest.class,
+                      TestRMMonitoring.class, TestRMNodeMonitoring.class,
+                      TestLocalInfrastructureRestartDownNodesPolicy.class, TestLocalInfrastructureStaticPolicy.class,
```

Comment:
As per the previous comment, I would remove here the `TestLocalInfrastructureRestartDownNodesPolicy.class`